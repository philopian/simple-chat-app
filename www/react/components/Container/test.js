import React from "react";
import Enzyme, { configure, shallow, mount, render } from "enzyme";
import renderer from "react-test-renderer";
import "raf/polyfill";
import Adapter from "enzyme-adapter-react-16";
import Component from "./index.js";

Enzyme.configure({ adapter: new Adapter() });

describe("Container (Snapshot)", () => {
  it("Container renders without crashing", () => {
    const component = renderer.create(<Component />);
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});
