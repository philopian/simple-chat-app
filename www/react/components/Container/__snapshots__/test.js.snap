// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container (Snapshot) Container renders without crashing 1`] = `
<div
  className="sc-bdVaJa gtxfII"
>
  <div
    className="sc-htpNat cXZGtB"
  >
    <p>
      Fake Chat app
    </p>
  </div>
  <div
    className="chat-box "
    id="chat-box"
  >
    <div
      className="sc-bxivhb gJPhtw"
    >
      <div
        className="arrow-right"
      />
      <div
        className="content padding-left"
      >
        <div
          className="username"
        >
          Some User
        </div>
        <div
          className="message"
        >
          ffffff
        </div>
        <div
          className="timestamp"
        >
          10:10pm
        </div>
        <div
          className="deal-with-float"
        />
      </div>
    </div>
  </div>
  <div
    className="footer"
    onClick={[Function]}
  >
    <div
      className="sc-ifAKCX jimocm"
    >
      <div
        className="input-section"
      >
        <input
          className="username"
          name="username"
          onChange={[Function]}
          type="text"
        />
        <button
          className="create-user-button"
          onClick={[Function]}
        >
          Send
        </button>
      </div>
    </div>
  </div>
</div>
`;
